### ISSUE TITLE
Churn interval shown instead pool activation interval

### ISSUE THREAD

**Issue by TreefeedXavier** [thorchain/asgardex-electron#1994]
## Expected Behavior

Blocks till pool activation shown

## Current Behavior

Blocks till node churn shown\wrong calculation

## Steps to Reproduce

1. Open pools - Pending
2. Observe Block left column

## Possible Solution

Change churn block left to pool activation blocks left

- ASGARDEX desktop version:

- Operating system and version:

## Your Environment - Development Version (skip if not using a development version)

0.6.0

- Operating system and version:

Windows 10

**Comment by veado** [comment]
## TL;DR 
No issue found. `left blocks` are based on  `POOLCYCLE` (Mimir or Midgard), but not on on churn interval.

## Details
Value for `blocks left` in pending pools are values based on `POOLCYCLE` (set by Mimir) or `PoolCycle` (defined in Midgard). It is computed as follow:

1. Grab `POOLCYCLE` from [Mimir](https://thornode.thorchain.info/thorchain/mimir) or [Midgard -> constants](https://midgard.thorchain.info/v2/thorchain/constants) (if Mimir is not set). 

   Code: https://github.com/thorchain/asgardex-electron/blob/c499590e2b2b36a97440cd3003302ed15dbd6e09/src/renderer/hooks/usePoolCycle.ts#L31-L70

2. Get latest block via [Midgard -> lastblock](https://midgard.thorchain.info/v2/thorchain/lastblock) 

   Code: https://github.com/thorchain/asgardex-electron/blob/8c0b51b1dc87292118906f1ad11f4c5dba88ef93/src/renderer/services/midgard/service.ts#L73-L105

3. Calculate blocks left based on `newPoolCycle - (lastHeight % newPoolCycle))` 

   Code: https://github.com/thorchain/asgardex-electron/blob/030e0e17f4b5f6284aa7bef682a1b615bf83a129/src/renderer/views/pools/Pools.utils.ts#L86-L101

### Hint

To compare `blocks left` for `pending pools` with `blocks left` for `churn interval`, check https://veado.github.io/thorchain-churn-countdown/
